ChronosTankDrive/src/main/java/frc/robot/subsystems/shooter/Hood.java

```diff
@@ -100,27 +99,34 @@ public void updateSmartDashboard() {
   public void init(){}
   public void periodic() {
     if (m_joystick.getPOV() == 90) {
```

Comment:
@TonyBaston - This only works for the XBox joystick. Are you planning to abstract this too? If not, is this required for operating the bot, or just for testing?

Reply:
This was just for testing